src/background/permissions.js

```diff
@@ -0,0 +1,52 @@
+import browser from "webextension-polyfill";
```

Comment:
as this file is used by both content-script and background, maybe we can move it out of the `background` folder?